paper/numbering.tex

```diff
@@ -4,24 +4,33 @@ \section{Numbering system}\label{numbering-app}
 
 For this formal definition we use the natural numbers $0,1,2,\dots$ to represent and order indeterminate variables; however, in the main text, we use the symbols $x,y,z,w,u,v,r,s,t$ instead (and do not consider any laws with more than eight variables).
 
-We extend the ordering on indeterminate variables to a well-ordering on words $w$ in the free magma generated by these variables by declaring $w > w'$ if one of the following holds:
+To define the ordering we use on equational laws, we first conside the case where there is a single indeterminate $\ast$.
+We place a well-ordering on words $w,w'$ with a single indeterminate $\ast$ by declaring $w > w'$ if one of the following holds:
 \begin{itemize}
     \item $w$ has a larger order than $w'$.
     \item $w = w_1 \op w_2$ and $w' = w'_1 \op w'_2$ have the same order $n \geq 1$ with $w_1 > w'_1$.
     \item $w = w_1 \op w_2$ and $w' = w'_1 \op w'_2$ have the same order $n \geq 1$ with $w_1 = w'_1$ and $w_2 > w'_2$.
 \end{itemize}
 Thus, for instance
-$$ 0 < 1 < 0 \op 0 < 0 \op 1 < 1 \op 0 $$
-and
-$$ 1 \op 1 < 0 \op (0 \op 0) < (0 \op 0) \op 0.$$
+$$ \ast < \ast \op \ast < \ast \op (\ast \op \ast) < (\ast \op \ast) \op \ast.$$
 
-We similarly place a well-ordering on equational laws $w_1 \formaleq w_2$ by declaring $w_1 \formaleq w_2 > w'_1 \formaleq w'_2$ if one of the following holds:
+We similarly place a well-ordering on equational laws $w_1 \formaleq w_2$ with a single indeterminate $\ast$ by declaring $w_1 \formaleq w_2 > w'_1 \formaleq w'_2$ if one of the following holds:
 as follows:
 \begin{itemize}
 \item  $w_1 \formaleq w_2$ has a longer order than $w'_1 \formaleq w'_2$.
 \item If $w_1 \formaleq w_2$ has the same order as $w'_1 \formaleq w'_2$, and $w_1 > w'_1$.
 \item If $w_1 \formaleq w_2$ has the same order as $w'_1 \formaleq w'_2$, $w_1 = w'_1$, and $w_2 > w'_2$.
 \end{itemize}
+Thus for instance
+$$ (\ast \op \ast \formaleq \ast \op (\ast \op \ast)) < (\ast \op \ast \formaleq (\ast \op \ast) \op \ast).$$
+
+Finally for equational laws with alphabet $x,y,z,w,u,v,r,s,t$, define the \emph{shape} of that law to be the law formed by replacing all indeterminates with $\ast$; for instance, the shape of \eqref{eq4512}, $(\ast \op \ast) \op \ast \formaleq \ast \op (\ast \op \ast)$, is $(\ast \op \ast) \op \ast \formaleq \ast \op (\ast \op \ast)$.  We then place a well-ordering $w_1 \formaleq w_2$ with indeterminates $x,y,z,w,u,v,r,s,t$ by declaring $w_1 \formaleq w_2 > w'_1 \formaleq w'_2$ if one of the following holds:
```

Comment:
I suppose the first `$(\ast \op \ast) \op \ast \formaleq \ast \op (\ast \op \ast)$` should be `$(0\op 1)\op 2=0\op (1\op 2)$`?